### ISSUE TITLE
New batch and activity records though no new data has been uploaded

### ISSUE THREAD

**Issue by jsayers-bsc** [MotusDev/Motus-TO-DO#926]
I've run tagme() on [this receiver](https://motus.org/data/receiver?id=1282) a few times in the last week. No new data has been uploaded since last fall and every time I make the call, I get the no new batch records, as expected. 

But this morning when I did it, there were 4 new batch records to check, and 6 new batch records to check for activity.

Why would there be new batch records to check if nothing new has been uploaded?

![image](https://user-images.githubusercontent.com/55001144/111346334-8021b680-8654-11eb-9fb2-0afc38b345c4.png)

**Comment by jsayers-bsc** [comment]
This morning several of the activity records were downloaded again.

![2021-03-17_095959](https://user-images.githubusercontent.com/55001144/111480534-7fe2f300-8708-11eb-9c94-ad607bbec112.jpg)

These 3 were in the list yesterday (including 1 additional record not present today). 

Previous to yesterday when I downloaded this receiver I got no new batch or activity records. 

Where are the new records coming from if no new data has been uploaded, or why are the same activity records downloaded today as yesterday?
